swatch-contracts/src/main/java/com/redhat/swatch/contract/resource/ContractsTestingResource.java

```diff
@@ -118,6 +119,24 @@ public Contract updateContract(String uuid, Contract contract)
     return service.updateContract(contract);
   }
 
+  @Override
+  @Transactional
+  @RolesAllowed({"test"})
+  public StatusResponse syncAllContracts() throws ApiException, ProcessingException {
+    log.info("Syncing All Contracts");
+    var contracts = service.getAllContracts();
+    for (ContractEntity org : contracts) {
```

Comment:
If the contracts list is empty then show the message "No active contract found for the orgIds"